content/sensu-go/5.2/reference/license.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "License management"
 linkTitle: "License Management"
-description: "Activate and manage your enterprise license for Sensu Go."
+description: "Sensu Go includes license-activated features designed for enterprises. Activate and manage your enterprise license for with sensuctl and your Sensu account. Read the reference documentation to learn more."
```

Comment:
```suggestion
description: "Sensu Go includes license-activated features designed for enterprises. Activate and manage your enterprise license with sensuctl and your Sensu account. Read the reference documentation to learn more."
```

Also the last sentence feels like there should be a link. Or is that pointing to read the rest of this page?

Reply:
+1 to your suggestion. For some background, having a call-to-action in these SEO descriptions is considered a best practice (per my conversations with GTM), even though there's no link.